app/assets/javascripts/shared/datatable/state.js

```diff
@@ -1,53 +1,61 @@
-$(document).on('preInit.dt', 'body.issues.index', function (e, settings) {
-  var $paths = $('[data-behavior~=datatable-paths]');
-  if ($paths.data('table-state-url') == undefined) {
-    return;
-  }
+$(document).on(
+  'preInit.dt',
+  'body.issues.index',
```

Comment:
Is this code in the right location? If it's tied to a page, then it shouldn't live under /shared/datable/ but under tylium/pages/

If this is really shared code across multiple DTs then we need to remove the hard-coding and use a data-behavior or some other trigger to hook into it.

Reply:
Apparently the hardcoding is not needed considering that we're already testing for the presence of the `state-url`